include/secp256k1_rangeproof.h

```diff
@@ -287,6 +296,32 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_rangeproof_info(
   size_t plen
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(3) SECP256K1_ARG_NONNULL(4) SECP256K1_ARG_NONNULL(5);
 
+/** Returns an upper bound on the size of a rangeproof with the given parameters
+ *
+ * An actual rangeproof may be smaller, for example if the actual value
+ * is less than both the provided `max_value` and 2^`min_bits`, or if
+ * the `exp` parameter to `secp256k1_rangeproof_sign` is set such that
+ * the proven range is compressed. In particular this function will always
+ * overestimate the size of single-value proofs. Also, if `min_value`
+ * is set to 0 in the proof, the result will usually, but not always,
+ * be 8 bytes smaller than if a nonzero value had been passed.
+ *
+ * The goal of this function is to provide a useful upper bound for
+ * memory allocation or fee estimation purposes, without requiring
+ * too many parameters be fixed in advance.
+ *
+ * To obtain the size of largest possible proof, set `max_value` to UINT64_MAX.
```

Comment:
Okay I get the point of the `value` now but then the line here in this PR is still somewhat unclear to the reader.

So "largest possible proof" here is really quantified over all values of `min_bits` because if `max_value == UINT64_MAX`, then `min_bits` really won't matter at all.

Maybe this could be clarified then:
```suggestion
 * To obtain the size of largest possible proof, set `max_value` to
 * `UINT64_MAX` (and `min_bits` to any valid value such as 0).  
```

Reply:
You've got it -- and yes, good idea, I'll update the docs.